samples/Microsoft.TestPlatform.TranslationLayer.E2ETest/Microsoft.TestPlatform.TranslationLayer.E2ETest.csproj

```diff
@@ -17,6 +17,7 @@
   <ItemGroup Condition=" '$(TargetFramework)' == 'net451' ">
     <ProjectReference Include="..\UnitTestProject\UnitTestProject.csproj" />
     <Reference Include="System" />
+    <Reference Include="System.Runtime" />
```

Comment:
Why is this?

Reply:
For unix, we are referring multitargeting pack to build desktop assemblies. It requires the dependencies to be explicitly called out.

Reply:
How can we make sure this for future commits?

Reply:
Unix build will break if we add a dependency and not explicitly mention it in csproj. It will fail the PR :)